manifests/local-storage.yaml

```diff
@@ -82,6 +82,18 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        resources:
+          requests:
+            memory: 30Mi
```

Comment:
Is there a reason to specify the request and limit for local path provisioner deployment?

Reply:
Yes, it uses the resources so I'd like to make sure they are available.